pygrocy/grocy.py

```diff
@@ -157,15 +157,39 @@ def stock(self, get_details: bool = False) -> List[Product]:
     def volatile_stock(self) -> CurrentVolatilStockResponse:
         return self._api_client.get_volatile_stock()
 
-    def expiring_products(self) -> List[ProductData]:
-        return self.volatile_stock().expiring_products
+    def expiring_products(self, get_details: bool = False) -> List[Product]:
+        raw_eg_product = self.volatile_stock().expiring_products
```

Comment:
Are `_eg_`, `_ei_`, `_m_  abbreviations for something I don't get? Please rename them to raw_expiring_products, raw_expired_products and raw_missing_products.